src/apps/testapps/testH3Line.c

```diff
@@ -64,7 +65,7 @@ static void h3Line_invalid_assertions(H3Index start, H3Index end) {
     int sz = H3_EXPORT(h3LineSize)(start, end);
     t_assert(sz < 0, "line size marked as invalid");
 
-    H3Index* line = {0};
+    H3Index *line = {0};
```

Comment:
I understand it's due to the linter, but I've never understood the C style to put the star next to the variable name. The type of the variable is a pointer to an H3Index, and you reference the variable with just the text part of the name, so the star seems to be more reasonably associated with that type.

Reply:
I think the reasoning for this linter rule is due to how C handles `H3Index *a, b;` which creates one pointer-to-H3Index and one H3Index.